exporters/otlp/metrics/src/main/java/io/opentelemetry/exporter/otlp/metrics/OtlpGrpcMetricExporterBuilder.java

```diff
@@ -7,6 +7,7 @@
 
 import static io.grpc.Metadata.ASCII_STRING_MARSHALLER;
 import static io.opentelemetry.api.internal.Utils.checkArgument;
+import static io.opentelemetry.exporter.otlp.internal.SslUtil.setTrustedCertificatesPem;
```

Comment:
I think I'd prefer to not have this method statically imported. I'd rather see the explicit call to `SslUtil.set...` for clarity at the callsite.